t04_refactored.py

```diff
@@ -231,8 +231,36 @@ def team_19_adv():
 ###################################################################################
 
 def team_20_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1aO8daq4WLqOsChlhvmClBHNZEx4faJcmsvGXPyOU990/edit?usp=sharing
+    :return:
+    America Gaona Borges
+    Jessica Douthit
+    """
+    global isDead
+    direction = input("Go with him further Up, or Down? ")
+
+    if direction == "Up":
+        print("You've reached a large vault... a large console stands to its right...")
+        print("The console lights up when you reach it... it asks for a passcode")
+        input("Guess the code" or "Ask for help? ")
+        if "Ask for help? ":
```

Comment:
This code never _doesn't_ run. It's not a comparison between two things, for example, `x == 1`. It's simply asking if the string `"Ask for help? "` exists, which it does, always, because... it is there.